articles/tutorials/step-up-authentication.md

```diff
@@ -5,71 +5,71 @@ description: You can add step-up authentication to your app with Authentication
+* Authentication Context Class Reference `acr`: string used to specify the authentication class performed on the current session. See `acr` under [ID Token](http://openid.net/specs/openid-connect-core-1_0.html#IDToken). Currently, Auth0 utilizes the [Multi-Factor Authentication policy](http://schemas.openid.net/pape/policies/2007/06/multi-factor).
 
-* `amr`- Authentication Methods References: is a JSON case senstitive string list of methods that were used to authenticate the current session.  For instance, values might indicate that both password and OTP authentication methods were used.  See the [Authentication Methods References](http://openid.net/specs/openid-connect-core-1_0.html) page for more details.
+* Authentication Methods References `amr`: JSON array of strings listing the methods used to authenticate the current session. For example, the `amr` might indicate that the current session was authenticated using a username/password.  See `amr` under [ID Token](http://openid.net/specs/openid-connect-core-1_0.html#IDToken).
 
-* `acr_values`: this is a space-separated string that specifies the `acr` values that have been requested to use for processing the request, with the values appearing in order of preference. This can be used to request the class of `acr` above when authentication is to be performed.  See [here](http://openid.net/specs/openid-connect-core-1_0.html) for more details.
+* `acr_values`: string specifying the `acr` values that have been user to process the request in order of preference. See `acr_values` under [Authentication Request](http://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint).
 
-`acr` and `amr` are both available on the [ID token](/tokens/id-token) of the current session, when appropriate. Both can signal to use MFA. The `acr_values` field is added to the request for authentication.
+When appropriate, both the `acr` and `amr` are available on the [ID token](/tokens/id-token) of the current session. You can use both to signal the need for MFA. The `acr-values` field is added to the authentication request.
 
 ## Example
 
-To request that Auth0 require a multifactor authentication, add the field `acr_values` to the authentication along with the `acr` level desired. For example, with [Auth0.js](/libraries/auth0js) it would work like the following code snippet:
+To request that Auth0 require multifactor authentication for certain situations, add the `acr_values` field to the authentication request along with the `acr` level desired. For example, if you're using the [auth0.js library](/libraries/auth0js), your sign in snippet might look something like this:
```

Comment:
Maybe rephrase `To request that Auth0 require multifactor authentication for certain situations` to something simpler like `To enable step-up-authentication`?